samples/tab-external-auth/csharp/TabExternalAuth/Views/Auth/GoogleEnd.cshtml

```diff
@@ -0,0 +1,35 @@
+﻿@{
+    ViewBag.Title = "title";
+    Layout = "_Layout";
+}
+
+ <script src="https://res.cdn.office.net/teams-js/2.0.0/js/MicrosoftTeams.min.js" crossorigin="anonymous"></script>
```

Comment:
Why old teams js version?

Reply:
authentication.authenticate is failing for lates teams js